### PR TITLE
Add figcd

### LINKED ISSUE #40: Add figcd
**Is your resource open source?**
Yes

**Is your request for a plugin, widget, or developer resource?**
Developer Resource

**What is the URL for your resource repo?**
https://github.com/parrot-global/figcd

### PR DESCRIPTION
**Pull request by acohen31**
Resolves #40

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,6 +132,18 @@ A collection of useful helper functions to import to your Figma plugin project
 
 ---
 
+
+**[⬆ Back to TOC](#table-of-contents)**
```
Comment (akbarbmirza) [inline README.md:136]: This Back to TOC should be below the figcd entry. Fix that and it'll be good to merge.

**Review by akbarbmirza — CHANGES_REQUESTED** [review]
this looks great - just need to adjust the TOC link location. Once that's fixed, I'll merge it.

**Comment by acohen31** [comment]
I made the change and added a subsection hook for the CI/CD Release Tools under the resource section. Let me know if there is anything else!

**Review by akbarbmirza — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -132,6 +133,18 @@ A collection of useful helper functions to import to your Figma plugin project
 
 ---
 
+### CI/CD Release Tools
+
+#### figcd
+
+[SOURCE CODE](https://github.com/parrot-global/figcd) · [MIT](https://github.com/evilmartians/figma-polychrom/blob/main/LICENSE)
```
Comment (akbarbmirza) [inline README.md:140]: Hey @acohen31 - I just realized that the license URL used here is wrong. Can you please fix this?

Comment (acohen31) [reply README.md:140]: Sure I'll get this done

**Pull request merged**